build.sh

```diff
@@ -440,14 +440,25 @@ __CleanBuild=0
 __VerboseBuild=0
 __SignTypeReal=""
 __CrossBuild=0
-__ClangMajorVersion=3
-__ClangMinorVersion=5
 __MSBuildPath=$__ProjectRoot/Tools/MSBuild.exe
 __NuGetPath="$__PackagesDir/NuGet.exe"
 __DistroName=""
 __cmakeargs=""
 __OfficialBuildIdArg=
 __SkipGenerateVersion=0
+__ClangMajorVersion=3
+__ClangMinorVersion=5
+while :; do
+    if [ $# -le 0 ]; then
+        break
+    fi
+    lowerI="$(echo $1 | awk '{print tolower($0)}')"
+    if [ $lowerI  == cross ]; then
```

Comment:
There are two reasons. First, I added this patch in front of the existing 'cross' case statement to handle many input cases as following: 
> time ./build.sh arm release clean cross
> time ./build.sh arm release clean clang3.8 cross 
> time ./build.sh arm release clean cross clang3.8
> time ./build.sh clang3.5 arm release clean cross 
> time ./build.sh arm release clean cross clang3.5

Second, I added this patch instead of  pattern matching using 'cross' case statement in order to handle possibility of similar arguments such as arm, arm-softfp in the future. Even though the arguments of build.sh will be increased gradually, the execution time will be within one second. 
> time ./build.sh arm release clean cross-armv8

So, at that time. I did not use the below 'case' method. If you think that  the 'cross' case  is better although we consider the above two situations, I will re-submit  the PR with  the 'cross' case.
> cross_arm=$@
> case $cross_arm in
>     *cross*) 
>         __ClangMajorVersion=3
>         __ClangMinorVersion=6
> esac 
>